### DISCUSSION TITLE
External preview 怎么使用电脑的vscode查看文件

### DISCUSSION THREAD

**Discussion by Masterlovsky** [AlistGo/alist#4178] [Q&A]
我按照说明文档的方法添加了 
 "/.*/": {
    "VSCode": "vscode://$url"
  }
点击右边的vscode能正常打开我电脑上的vscode，但是alist上的文件没有传递过来。请问我哪里设置的不对吗？

**Comment by xhofe** [comment]
这里我只是举例子，实际的vscode可能并不支持通过url打开文件，需要你自己查找相关资料。

**Reply by Masterlovsky** [reply]
> 这里我只是举例子，实际的vscode可能并不支持通过url打开文件，需要你自己查找相关资料。

vscode支持通过链接打开链接，可以使用vscode://file/$url 这样，但是我获取到的url前面会多一个/p/，而且vscode会报这个链接打不开，链接里面是\而不是/，不知道是不是这个windows路径的原因，作者大佬有思路不

**Reply by YowFung** [reply]
@Masterlovsky 大佬有找到解决方法了吗？
